src/publisher/services/publisher-api.ts

```diff
@@ -333,15 +333,6 @@ export class PublisherApi {
     });
   }
 
-  public async confirmDataTable(datasetId: string, revisionId: string): Promise<DataTableDto> {
-    logger.debug(`Confirming data table import for revision: ${revisionId}...`);
-
-    return this.fetch({
-      url: `dataset/${datasetId}/revision/by-id/${revisionId}/data-table/confirm`,
-      method: HttpMethod.Patch
-    }).then((response) => response.json() as unknown as DataTableDto);
-  }
```

Comment:
This was a no-op on the backend, it doesn't actually do anything other than return an object that we then don't even assign to a var on the frontend. I assume it was previously doing something more than this and as code was removed it became redundant.